i18n/management/commands/gather_i18n_strings.py

```diff
@@ -25,3 +26,11 @@ def handle(self, *args, **options):
                 outpath = os.path.abspath(os.path.join(source_dir, model.__name__ + ".json"))
                 with open(outpath, 'w') as outfile:
                     json.dump(strings, outfile, indent=4, sort_keys=True)
+
+        # Gather strings for translation in html files.
+        # For some reason makemessages won't create the locale/ directory but will create
+        # the subdirectories
+        subprocess.call(["mkdir", "locale/"])
+        subprocess.call(["python", "manage.py", "makemessages", "-l", "en"])
```

Comment:
We can use [`django.core.management`](https://docs.djangoproject.com/en/1.8/ref/django-admin/#running-management-commands-from-your-code) rather than `subprocess` for calling the manage.py commands

Reply:
Done.